infores_catalog.yaml

```diff
@@ -855,7 +855,7 @@ information_resources:
       - https://github.com/NCATSTranslator/Translator-All/wiki/ChEBI
     synonym:
       - ChEBI
-    knowledge level: curated
+    knowledge level: mixed
```

Comment:
Our team is using CHEBI's xrefs to Rhea and Reactome. 

Those xrefs are automatically-generated, according to the [Reference Manual](https://docs.google.com/document/d/13_FqP_8qkDdw5RFv5qMYo7xRyfY4xUiuVtLn845A6Io/edit#heading=h.5drhv33udo49) linked in their ["Sources" info](https://www.ebi.ac.uk/chebi/aboutChebiForward.do#Sources). 

But I also saw that CHEBI uses a lot of data sources and calls them "manual" in the Reference Manual, which might mean "curated"? 

So that's why I picked "mixed": that some edges would be "curated" and others like the ones we use would be automatically-generated...

Reply:
Given our convo below ([my question](https://github.com/biolink/biolink-model/pull/1429#issuecomment-1857114150), [Sierra's response](hhttps://github.com/biolink/biolink-model/pull/1429#issuecomment-1857154333)), I changed the knowledge level back to `curated`. So I think this is resolved...